pages/stable-version.html

```diff
@@ -93,13 +93,14 @@ <h4><a href="#releasenotes">Release notes</a></h4>
                 <li><a href="../doc/2021.02/release-notes-uyuni-server.html">Server</a></li>
```

Comment:
Maybe you can consider to make this more prominent, while you keep the **Important** without using capital letters. That  could be a good comprimise.

For example:
```
<ul>
    <li><a href="../doc/2021.02/release-notes-uyuni-server.html">For Uyuni Server</a></li>
    <li><a href="../doc/2021.02/release-notes-uyuni-proxy.html">For Uyuni Proxy</a></li>
<li>
<p><strong>Important:</string>Remember to review the release notes! They contain valuable information and, depending on your current version, required procedures to update to the most recent version.</p>
```